docs/Reference.md

```diff
@@ -1385,6 +1385,31 @@ Example:
 <WebView enableApplePay={true} />
 ```
 
+### `menuItems`
+
+An array of custom menu items that will be appended to the UIMenu that appears when selecting text (will appear after 'Copy' and 'Share...').  Used in tandem with `onCustomMenuSelection`
+
+Example:
+
+```javascript
+<WebView menuItems={['Tweet', 'Save for later']} />
+
+```
+
+### `onCustomMenuSelection`
+
+Function called when a custom menu item is selected.  It receives a Native event, which includes two custom keys: `customMenuKey` and `selectedText`.
```

Comment:
Small detail, but with localization it could be nice if the menu `key` and `label` were separate things.

Reply:
I meant the array of objects approach. IMO it makes the API clearer... usually, in my experience, IDs and UI text are not the same. The label might be a bunch of different things depending on the language, which with the "array of strings" approach means your code needs to be making sure to perform logic (e.g. switch statements, etc) with the localized strings, and logging could get confusing (might need to write logic to translate back from the display string into a string that can be logged in the developer's language), etc...

Reply:
Sorry for interrupting :), what do you think we use this array?

```
const menuItems = [{
    title: 'Highlight',
    onPress: () => {}
  },
  {
    title: 'Share',
    onPress: : () => {}
  },
  {
    title: 'Copy',
    onPress: : () => {}
  }
]
```

Reply:
@phuongwd I couldn't find a way to include a function in within an object and pass it to objective-c.

@TheAlmightyBob Updated this to include to keys -- `key` and `label`.  I think this PR should be ready for review now 🤞